ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -796,16 +822,16 @@ class OverflowAvatarView: AvatarView {
         var avatarFrame = bounds
         if hasBorder {
             let borderOffset = Constants.borderSize - borderWidth
-            avatarFrame.origin.x -= borderOffset
-            avatarFrame.origin.y -= borderOffset
-            avatarFrame.size.width += -borderOffset * 2
-            avatarFrame.size.height += -borderOffset * 2
+            avatarFrame.origin.x -= borderOffset * 1.75
```

Comment:
where is this 1.75 coming from?

Reply:
It was to adjust for the new inside border; replaced with existing variables for better readability